content/influxdb/cloud/reference/release-notes/cloud-updates.md

```diff
@@ -14,6 +14,39 @@ aliases:
 InfluxDB Cloud updates occur frequently. Find a compilation of recent updates below.
 To find information about the latest Flux updates in InfluxDB Cloud, see [Flux release notes](/influxdb/cloud/reference/release-notes/flux/).
 
+## June 2022
+
+### Private InfluxDB Cloud offering
+
+New support for private InfluxDB Cloud offering for annual customers. Please contact sales@influxdata.com for more information.
+
+### Sample IoT application and code snippets for Node.js
+
+Quickly start writing and querying your data with **JavaScript/Node.js**. Now, when you complete the **JavaScript/Node.js** onboarding wizard, you'll have the option to do either of the following:
+- View an IoT [**Sample App**](https://github.com/influxdata/iot-api-js) written in Node.js
+- View [**Boilerplate Snippets**](https://github.com/influxdata/nodejs-samples/) of code in Node.js
+
+### Help bar updates
+
+- If you have a [Usage-Based plan](/influxdb/cloud/account-management/pricing-plans/#usage-based-plan), you can now submit **Support requests** and questions directly to our **Salesforce Support queue** (without needing a Salesforce account). You'll receive a confirmation email with your ticket number for reference.
```

Comment:
to be sure, @Luckette --is this only for customers with Usage-Based plans? (I see the option to Contact Support, so maybe we have a Usage-Based plan)